src/createAnimatedComponent.js

```diff
@@ -253,6 +253,11 @@ export default function createAnimatedComponent(Component) {
       } else {
         // hostInstance can be null for a component that doesn't render anything (render function returns null). Example: svg Stop: https://github.com/react-native-svg/react-native-svg/blob/develop/src/elements/Stop.tsx
         const hostInstance = RNRenderer.findHostInstance_DEPRECATED(this);
+        if (!hostInstance) {
+          throw new Error(
```

Comment:
This is making using SVG `Stop` animations not work anymore. Simply removing this makes it work for me again, but I don't know if it will break other things